src/misc/land-boats.ts

```diff
@@ -0,0 +1,87 @@
+import { Material, TreeSpecies } from 'org.bukkit';
+import { BlockFace } from 'org.bukkit.block';
+import { Boat, EntityType, Player } from 'org.bukkit.entity';
+import { Action } from 'org.bukkit.event.block';
+import { PlayerInteractEvent, PlayerJoinEvent } from 'org.bukkit.event.player';
+import { VehicleEnterEvent } from 'org.bukkit.event.vehicle';
+import { ItemStack } from 'org.bukkit.inventory';
+
+const Boaters = new Set<Player>();
```

Comment:
Use camelCase (not CapitalCamelCase) for mutable global variables.
```suggestion
const boaters = new Set<Player>();
```